code/datums/traits/negative.dm

```diff
@@ -11,6 +11,17 @@
 	lose_text = "<span class='notice'>You feel awake again.</span>"
 	medical_record_text = "Patient has abnormal sleep study results and is difficult to wake up."
 
+/datum/trait/brainproblems
+	name = "Brain Tumor"
+	desc = "You have a little friend in your brain that is slowly destroying it. Better bring some mannitol!"
+	value = -2
+	gain_text = "<span class='danger'>You feel smooth.</span>"
```

Comment:
Okay we get it armhulenn don't need to tell us about yourself too much

Reply:
Kevin is a smoothbrain